neo4j/conf.yaml.example

```diff
@@ -0,0 +1,12 @@
+init_config:
+    default_timeout: 5
+
+instances:
+    -   url: http://ec2-54-84-9-93.compute-1.amazonaws.com:7474/
+        port: 7474
+        server: http://ec2-54-84-9-93.compute-1.amazonaws.com
```

Comment:
Should this be neo4j_server? Could  we change the example from this EC2 host name to something that is more clearly an example? Finally, it looks like server and url might be duplicates?